pkgs/tools/nix/nixos-render-docs/src/nixos_render_docs/manpage.py

```diff
@@ -130,7 +132,8 @@ def softbreak(self, token: Token, tokens: Sequence[Token], i: int, options: Opti
         return " "
     def code_inline(self, token: Token, tokens: Sequence[Token], i: int, options: OptionsDict,
                     env: MutableMapping[str, Any]) -> str:
-        return _protect_spaces(man_escape(token.content))
+        s = _protect_spaces(man_escape(token.content))
+        return f"\\fR\\(oq{s}\\(cq\\fP" if self.inline_code_is_quoted else s
```

Comment:
Maybe mention that manpages do not support inline code also in a comment so that people reading the code do not have to be intimately familiar with the format and use blame?